miio/integrations/vacuum/dreame/dreamevacuum_miot.py

```diff
@@ -66,25 +157,38 @@ class OperatingMode(Enum):
     ZonedCleaning = 19
 
 
-class FaultStatus(Enum):
+class FaultStatus(FormattableEnum):
     Unknown = -1
     NoFaults = 0
 
 
-class DeviceStatus(Enum):
+class DeviceStatus(FormattableEnum):
     Unknown = -1
     Sweeping = 1
     Idle = 2
     Paused = 3
     Error = 4
     GoCharging = 5
     Charging = 6
+    Mopping = 7
     ManualSweeping = 13
 
 
+class WaterFlow(FormattableEnum):
+    Unknown = -1
+    Low = 1
+    Medium = 2
+    High = 3
+
+
+def _enum_as_dict(cls):
+    return {x.name: x.value for x in list(cls)}
+
+
 class DreameVacuumStatus(DeviceStatusContainer):
-    def __init__(self, data):
+    def __init__(self, data, model):
         self.data = data
+        self.model = model
```

Comment:
Would you mind adding a docstring containing the contents of the input data?

Reply:
Oh, I meant the data that gets passed in, to help the future readers to understand the contents. See https://github.com/rytilahti/python-miio/blob/master/miio/integrations/humidifier/deerma/airhumidifier_jsqs.py#L47 for an example.

Reply:
Alternatively, as the tests contains the raw payloads, simply remove this docstring.